src/assets/_project/_blocks/components/site-search/qg-site-search.js

```diff
@@ -135,66 +131,71 @@ $(function () {
   };
 
   qgSiteSearch.fn.keyboardNavigation = function (event) {
+    var self = $(this);
     var keyCode = event['keyCode'];
-    var focusableList = $('.qg-search-concierge.show').find('a, button');
-    var currentIndex = $('.qg-search-form').attr('data-navindex');
+    var focusableList = self.parents('.qg-site-search__form').find($('.qg-search-concierge.show')).find('a, button');
+    var currentIndex = self.parents('.qg-site-search__form').attr('data-navindex');
 
     if (keyCode === 40 && focusableList.length > currentIndex - 1) {
       currentIndex++;
       focusableList[currentIndex].focus();
-      $('.qg-search-form').attr('data-navindex', currentIndex);
+      self.parents('.qg-site-search__form').attr('data-navindex', currentIndex);
     } else if (keyCode === 38) {
       if (currentIndex > 0) {
         currentIndex--;
         focusableList[currentIndex].focus();
-        $('.qg-search-form').attr('data-navindex', currentIndex);
+        self.parents('.qg-site-search__form').attr('data-navindex', currentIndex);
       } else {
-        $('.qg-search-form input[type=text].form-control').focus();
+        self.parents('.qg-site-search__form').find($('input[type=text].form-control')).focus();
       }
     }
   };
 
   // Handle clearing the input field via button
   qgSiteSearch.fn.clearInputField = function (event) {
-    var clearButton = $('.qg-search-close-concierge');
-    var searchInput = $('#qg-search-query');
+    let inputTarget = event.target;
+    var clearButton = $(this).parent().find($('.qg-search-close-concierge'));
+    var searchInput = $(this).parent().find($('.qg-search-site__input'));
     searchInput.val('');
 
     // Remove the button
     clearButton.addClass('hide');
 
     // Close the concierge panels
-    qgSiteSearch.fn.closeConciergePanels();
+    qgSiteSearch.fn.closeConciergePanels(inputTarget);
   };
 
   // Handle selecting a suggestion
   qgSiteSearch.fn.searchSuggestionClick = function (event) {
     var targetElement = $(event['currentTarget']);
     var suggestionValue = targetElement.text();
-    var searchInput = $('#qg-search-query');
+    var searchInput = $('.qg-search-site__input');
```

Comment:
is this variable used?

Reply:
I have changed this to a class-based selector so that we can have multiple searches on a single page. I have made this change in the site-search partial HTML file.